frontend/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hmccaa/hmcc-web:gatsby-0.1.1
+FROM hmccaa/hmcc-web:gatsby-0.1.0-arm
```

Comment:
Revert this change. we do want want this to change because different people use different os/cpu platform

Reply:
Yes, unfortunately I haven't figured out a way to fix this yet :/ let's keep it as just the non-arm one for now...

@graceschae feel free to also put frontend/Dockerfile and cms/Dockerfile into the .gitignore as well so nobody else has to deal with this!